Makefile

```diff
@@ -76,19 +76,41 @@ ctl:
 run:
 	cargo run --no-default-features --features  "${ENABLE_FEATURES}" --bin tikv-server
 
+# An optimized build suitable for development and benchmarking
 release:
+	make dist_release
+
+# An optimized build with additional SSE4.2 optimizations
+unportable_release:
+	make dist_unportable_release
+
+# An optimized build with jemalloc memory profiling enabled
+prof_release:
+	make dist_prof_release
+
+# An optimized build instrumented with failpoints
+fail_release:
+	make dist_fail_release
+
+# The target used by CI/CD to build the distributable release artifacts.
+# Individual developers should only need to use the `dist_` rules when working
+# on the CI/CD system.
+dist_release:
 	cargo build --no-default-features --release --features "${ENABLE_FEATURES}"
 	@mkdir -p ${BIN_PATH}
 	@cp -f ${CARGO_TARGET_DIR}/release/tikv-ctl ${CARGO_TARGET_DIR}/release/tikv-server ${CARGO_TARGET_DIR}/release/tikv-importer ${BIN_PATH}/
 	bash scripts/check-sse4_2.sh
 
-unportable_release:
+# Distributable bins with SSE4.2 optimizations
+dist_unportable_release:
 	ROCKSDB_SYS_PORTABLE=0 make release
 
-prof_release:
+# Distributable bins with jemalloc memory profiling
+dist_prof_release:
 	ENABLE_FEATURES=mem-profiling make release
 
-fail_release:
+# Distributable bins instrumented with failpoints
+dist_fail_release:
```

Comment:
I have a question. When will failpoint release being useful? @BusyJay

Reply:
For Schrodinger